app/content/blog/building-beyond-beginner-guitar.md

```diff
@@ -0,0 +1,248 @@
+
+Focused around aspiring guitarists who went through some sort of 'guitar basics' lessions before stalling out on the question, "What do I do next?", the guitar course is built in an Urbit app called [Hawk](https://hawk.computer/~~/overview/). Hawk allows you to build powerful and flexible web pages served from your Urbit to the clearweb, and it is an excellent platform on top of which I can build my own little digital business. In this article, we'll cover:
+
+- How 'Beyond Beginner Guitar' works today
+- The external dependencies on which it depends
+- My dream for a 100% urbit-native future
+- A bit of detail on what it will take to get there
+
+## An Urbit Powered Business. Today.
+As an artist and creator, drawing people towards my work is a critical effort. While my Urbit acts as the central server for my course, the place all students come for the lessons, I maintain a presence in the external world by way of [my YouTube channel](https://www.youtube.com/@brbenji) where I share videos on both Urbit and music. Drawing a connection between this MEGACORP domain, and the sovereign landscape of my Urbit server is a necessary function in today's world.
+
+![beyond beginner guitar pipeline wireframe](https://s3.howm.art/uploads/bbg-pipeline.excalidraw.png)
+
+These videos are the main way students discover the guitar course. From the descriptions of my videos, and often as an included call to action in the video itself, I send people to a page hosted by my urbit where I can collect their emails. I also use this link to do a small bit of metric-tracking, to understand which videos are attracting new students to sign-up.
+
+Once the email is collected, I use it to communicate with them. This external dependency on email is a compromise, but in our current world not everyone can be assumed to have an Urbit so it is a necessary tradeoff. A second compromise is that I use Google forms in a followup flow to learn a little bit more about them and whether they are a good fit for the guitar course. This part could easily be done in an Urbit native manner in the near future. If they are, I send them a link, which brings them back another page served by my Urbit, the pay page.
+
+On the pay page, there are instructions for manual crypto payments:
+1. They pay my ethereum account the proper amount
+2. They copy the transaction hash
+3. They send it to me over email
+4. I verify it and send them the secret URL for the course.
+
+But, the primary flow uses Stripe for credit card payments. This is another entry point into the legacy world that is presently necessary, largely given the reliability of asusming someone who wants to improve their guitar skills has a credit card. The pay page includes an option to buy the course via a Stripe embed. Clicking the Buy button sends them to a secure, Stripe checkout.
```

Comment:
The primary flow, however, uses Stripe for credit card payments. This is another entry point into the legacy world that is presently necessary, largely because we can reliably assume someone who wants to improve their guitar skills has a credit card. The pay page includes a Stripe embed. Clicking the Buy button sends them to a secure, Stripe checkout.

- assume
- minor tweaks to sentences